queries.sql

```diff
@@ -7,4 +7,69 @@ SELECT date_of_birth FROM animals  WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = 't';
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+BEGIN TRANSACTION;
+SAVEPOINT SP1;
+UPDATE animals
+SET species = 'undefined';
+ROLLBACK TO SP1;
+
+BEGIN TRANSACTION;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon%';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
+
+BEGIN TRANSACTION;
+
+SAVEPOINT SP2;
+
+DELETE FROM animals;
+
+ROLLBACK
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235708707-fe26147d-aa87-40e4-be54-0df0c28c28cc.png)

- [ ] Kindly verify the changes using Select statements as required by the project requirements 😊 Which you can see above ⬆️

```SQL
   BEGIN;

   DELETE FROM animals WHERE ;
   SELECT COUNT(*) FROM ANIMALS;

   ROLLBACK;
   SELECT COUNT(*) FROM ANIMALS;
```